public/assets/.sprockets-manifest-83f48bb0c8ae8407f037eaa53e156bf0.json

```diff
@@ -0,0 +1 @@
c6e02eb7a533bf8a771b80c0ed996d67bf35ce10ac.css","users.css":"users-9996b4094906a62a0bb73a6a163e0cb46086315b8a1420f025b0fe8b63741c2b.css","application.js":"application-c298fdf2cd5aa1047c1565266a9064444a31fa7a05c4d84156f61966328d187a.js","controllers/application.js":"controllers/application-6ba512835f5e4f3393509c4a8b146b1b4f25889d3808218eb3fe88a66fec647d.js","controllers/hello_controller.js":"controllers/hello_controller-549135e8e7c683a538c3d6d517339ba470fcfb79d62f738a0a089ba41851a554.js","controllers/index.js":"controllers/index-31a9bee606cbc5cdb1593881f388bbf4c345bf693ea24e124f84b6d5c98ab648.js","controllers/search_controller.js":"controllers/search_controller-190ca2c0669027d45a9abe93ccce6157d907588d3d196ba68a71ef6ed9828ee4.js","controllers/test_card_controller.js":"controllers/test_card_controller-cabbea434aa0c392a5f7b41d5f2aa58ea5cfcbd9635711d0e2d3e1056a5f7eee.js","controllers/test_grouping_card_controller.js":"controllers/test_grouping_card_controller-4a3e2ca8cdeb732a664b54e5ead96c1eac3ee012620d508664d8edc2faf470e1.js","views/assignments/show.js":"views/assignments/show-8c3b63b88fa464fffb7ac6bfff6665b4148e7b3ffa114164b5e17738d41bd3f7.js","@rails--ujs.js":"@rails--ujs-8fbd650218b6b146a78aa5d7d6cd8eb8d9da87cc6d504aab7fc3c6b64e94d34b.js","turbo.js":"turbo-8b77232adadeefbcf302ffbec6078486a9b0bf62de4026e9816ef85aeb35dcd0.js","turbo.min.js":"turbo.min-d249cb94aaf5d33cdfcc830d92ca38f12344e8fc9718ac0e17fef7214cb3ea76.js","turbo.min.js.map":"turbo.min.js-bd61e9cdef5afbd125bfd0e002da3be63c5556c30a6d2d23064033e0d3827ef9.map","actiontext.js":"actiontext-78de0ebeae470799f9ec25fd0e20ae2d931df88c2ff9315918d1054a2fca2596.js","actiontext.esm.js":"actiontext.esm-328ef022563f73c1b9b45ace742bd21330da0f6bd6c1c96d352d52fc8b8857e5.js","trix.js":"trix-686ab55c2aea8035a7f728b61ec7afedfe857f70d6279ab453da775b7469e9e8.js","trix.css":"trix-5fc7656c4bff8fe505ff90fda4bc9409db4447ada6efcc1204914dc782c6066c.css","stimulus.js":"stimulus-f75215805563870a61ee9dc5a207ce46d4675c7e667558a54344fd1e7baa697f.js","stimulus-autoloader.js":"stimulus-autoloader-c584942b568ba74879da31c7c3d51366737bacaf6fbae659383c0a5653685693.js","stimulus-importmap-autoloader.js":"stimulus-importmap-autoloader-db2076c783bf2dbee1226e2add52fef290b5d31b5bcd1edd999ac8a6dd31c44a.js","stimulus-loading.js":"stimulus-loading-3576ce92b149ad5d6959438c6f291e2426c86df3b874c525b30faad51b0d96b3.js","stimulus.min.js":"stimulus.min-600d49735c78bb3e7920c6de98cf5bd7157039525565153b753fa610a6ab334e.js","stimulus.min.js.map":"stimulus.min.js-2cc63625fa177963b45da974806e7aee846cbf1d4930815733d0fdf3fb232325.map","activestorage.js":"activestorage-503a4fe23aabfbcb752dad255f01835904e6961d5f20d1de13987a691c27d9cd.js","activestorage.esm.js":"activestorage.esm-b3f7f0a5ef90530b509c5e681c4b3ef5d5046851e5b70d57fdb45e32b039c883.js","actioncable.js":"actioncable-323d156a10054ee2c94ba8a17c20458f542a17234f934ea53d16e0e1899fda5e.js","actioncable.esm.js":"actioncable.esm-b66d83871f0a453b10ba8b4c147c2bafa655caaa5d13fa34e9bf361c1c2aedb7.js"}}
```

Comment:
We should no be pushing `public/assets`. Please remove this and add it back to the `.gitignore`